parser/diff.go

```diff
@@ -0,0 +1,130 @@
+package parser
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/reviewdog/reviewdog/diff"
+	"github.com/reviewdog/reviewdog/filter"
+	"github.com/reviewdog/reviewdog/proto/rdf"
+)
+
+var _ Parser = &DiffParser{}
+
+type DiffParser struct {
```

Comment:
**[golint-circleci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
exported type DiffParser should have comment or be unexported